benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vp.prm

```diff
@@ -0,0 +1,196 @@
+# for uniformly refined meshes of level 3 and up. The velocity on
+# the boundaries is 5e-9 m/s, giving a background strain rate of
+# 1e-13 1/s. The weak medium and weak inclusion have the following
+# material properties: viscosity = 1e17 Pa s, cohesion = 1e7 Pa,
+# and internal friction angle = 0. The visco-plastic block has
+# the following material properties: viscosity = 1e23 Pa s, 
+# cohesion = 1e8 Pa, and internal friction angle = 37 degrees.
+# Gravity is set to zero and the average pressure is required
+# to be equal to zero. 
+
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Newton Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 500
+set CFL number                             = 0.5
+set Maximum time step                      = 20
+set Output directory                       = output_gerya_2019_vp
+set Timing output frequency                = 1
+set Pressure normalization                 = volume
+
+# Solver parameters
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block AMG
+    set Linear solver tolerance = 1e-7
+    set Number of cheap Stokes solver steps = 200
+  end
+  subsection Newton solver parameters
+    set Max Newton line search iterations        = 0
+    set Max pre-Newton nonlinear iterations      = 500 # Ensures only defect-correction Picard method is used
+    set Maximum linear Stokes solver tolerance   = 1e-2
+    set Nonlinear Newton solver switch tolerance = 1e-5
+    set SPD safety factor                        = 0.9
+    set Stabilization preconditioner             = SPD
+    set Stabilization velocity block             = SPD
+    set Use Newton failsafe                      = false
+    set Use Newton residual scaling method       = false
+    set Use Eisenstat Walker method for Picard iterations = true
+  end
+end
+
+
+# Model geometry (100x100 km, initial 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 2
+    set Y repetitions = 2
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Globally refine the mesh. A minimum of 3 global refinements
+# is needed to resolve the edges of the weak viscous inclusion.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 5
+  set Time steps between mesh refinement = 0
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+# Velocity boundary conditions (5e-9 m/s on each boundary)
```

Comment:
This is just a thought for next time (you don't actually have to change anything). But if you want to prescribe the velocity in m/s, why didn't you just set Use years in output instead of seconds to false? The strain rate output won't change.

Reply:
Yep, removed!